lesson3/Student.java

```diff
@@ -0,0 +1,17 @@
+package lesson3;
+
+import java.util.List;
+
+public record Student(String name, int age, List<Grade> listGradle, String school, Address address) {
+
+    @Override
+    public String toString() {
+        return "Student{" +
```

Comment:
Just an advice to use in future: do not use `public String toString()` as a part of business logic.